src/abstractdatatable/join.jl

```diff
@@ -3,14 +3,17 @@
 ##
 
 # Like similar, but returns a nullable array
-similar_nullable{T}(dv::AbstractArray{T}, dims::Union{Int, Tuple{Vararg{Int}}}) =
```

Comment:
Right, sorry. One of these constructors started throwing deprecation warning since we started this PR